src/parse-options.js

```diff
@@ -13,6 +17,13 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
```

Comment:
We should add some tests for this logic perhaps [here](https://github.com/NickTomlin/protractor-flake/blob/master/test/unit/index.test.js#L167)